src/Compilers/CSharp/Portable/Generated/CSharp.Generated.g4

```diff
@@ -22,6 +22,18 @@ identifier_name
   | identifier_token
   ;
 
+attribute_list
```

Comment:
nit: do we know why this got moved by the generator?

Reply:
The generator does a DFA walk through the constructs, writing them out in that order.  So it's most likely that there's a way to walk from `type` to `attribute_list`, which doesn't exist just with `name`.